src/main/java/com/froobworld/farmcontrol/utils/EntityTypeUtils.java

```diff
@@ -16,6 +17,10 @@ public static Predicate<SnapshotEntity> fromString(String string) {
                 return entity -> Animals.class.isAssignableFrom(entity.getEntityClass());
             } else if (category.equalsIgnoreCase("monster")) {
                 return entity -> Monster.class.isAssignableFrom(entity.getEntityClass());
+            } else if (category.equalsIgnoreCase("projectile")) {
```

Comment:
> 

Players can create infinite amount of projectile entities like snowball or ender pearl via bubble column (soul sand + water)

Reply:
For some reason if i write in group "snowball" or "ender_pearl" is doesn't work

Reply:
The reason `snowball` and `ender_pearl` don't work is because they're not mobs (a full list of mobs can be found here https://hub.spigotmc.org/javadocs/spigot/org/bukkit/entity/Mob.html)

Adding a projectile category won't change this - it will also do nothing because the plugin doesn't support anything that isn't a mob